Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml

```diff
@@ -63,6 +63,15 @@
   - type: NpcFactionMember
     factions:
     - PetsNT
+  - type: Sprite
+    sprite: DeltaV/Mobs/Pets/lawyercarp.rsi # Fancy Fishe
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: alive
+      - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+        state: mouth
```

Comment:
```suggestion
       - state: mouth
```

Reply:
to set shader to unshaded do `shader: unshaded` i dont think that layer exists